src/c4/C4_MPI.t.hh

```diff
@@ -230,6 +230,17 @@ int gatherv(T *send_buffer, int send_size, T *receive_buffer, int *receive_sizes
   return Result;
 }
 
+//------------------------------------------------------------------------------------------------//
+template <typename T>
```

Comment:
Could you add a doxygen comment block here? Thanks!

Reply:
You will also need to add a stub with this signature to `C4_Serial.t.hh`.

Reply:
Sorry for the delay - will get these modifications in shortly.

Reply:
@alexrlongne Added a doxygen comment block to the corresponding place in `C4_Functions.hh` (it seems this is where the other comment blocks are).
@KineticTheory Added a serial stub version of this function in `C4_Serial.t.hh`.